x/dispensation/Steps-Investor.md

```diff
@@ -0,0 +1,56 @@
+# Steps to follow by Investor/Distributor 
```

Comment:
in general just tidy up the punctuation below "1. " and commas, <-space after, not before